projectdo

```diff
@@ -187,6 +187,38 @@ try_lein() {
   fi
 }
 
+# justfile
+
+has_just_target() {
+  target="${1?}"
+  output=$(just -n "${target}" 2>&1)
+  exit_code=$?
+  return $exit_code
+}
+
+try_justfile() {
+  if [ -f justfile ]; then
+    if ! has_command "just"; then
+      echo "Found a justfile but 'just' is not installed."
+      exit 1
+    fi
+    if $IS_TOOL || [ "$ACTION" = build ]; then
+      execute "just build"
```

Comment:
When `IS_TOOL` is `true` only the name of the command should be run. You probably looked at the implementation for `make`, but that is a bit of a special case since building with `make` is just `make` (and not, say, `make build`) 😉 

This allows for `alias p='projectdo tool'` and then doing stuff like `p anything` which will then evoke `just anything`.

The easiest way to implement this is to use `execute_command` which handles this special case. With that change I think using a `case` statement like some of the other tools would make sense as well.

Reply:
Ah, I see now. I think with `execute_command` the `case` statement isn't required, or perhaps I missed something?

Reply:
Thanks. And yes, with the current implementation `case` is not needed :)